shablona/shablona.py

```diff
@@ -1,211 +1,78 @@
-from __future__ import absolute_import, division, print_function
 import numpy as np
-import pandas as pd
-import scipy.optimize as opt
-from scipy.special import erf
-from .due import due, Doi
 
-__all__ = ["Model", "Fit", "opt_err_func", "transform_data", "cumgauss"]
+__all__ = ['midi_to_hz', 'hz_to_midi', 'hz_to_period']
 
-
-# Use duecredit (duecredit.org) to provide a citation to relevant work to
-# be cited. This does nothing, unless the user has duecredit installed,
-# And calls this with duecredit (as in `python -m duecredit script.py`):
-due.cite(Doi("10.1167/13.9.30"),
-         description="Template project for small scientific Python projects",
-         tags=["reference-implementation"],
-         path='shablona')
-
-
-def transform_data(data):
+def midi_to_hz(notes):
     """
-    Function that takes experimental data and gives us the
-    dependent/independent variables for analysis.
-
-    Parameters
-    ----------
-    data : Pandas DataFrame or string.
-        If this is a DataFrame, it should have the columns `contrast1` and
-        `answer` from which the dependent and independent variables will be
-        extracted. If this is a string, it should be the full path to a csv
-        file that contains data that can be read into a DataFrame with this
-        specification.
-
-    Returns
-    -------
-    x : array
-        The unique contrast differences.
-    y : array
-        The proportion of '2' answers in each contrast difference
-    n : array
-        The number of trials in each x,y condition
+    Hello Part 6!  You should add documentation to this function.
     """
-    if isinstance(data, str):
-        data = pd.read_csv(data)
 
-    contrast1 = data['contrast1']
-    answers = data['answer']
+    return 440.0 * (2.0 ** ((np.asanyarray(notes) - 69.0)/12.0))
 
-    x = np.unique(contrast1)
-    y = []
-    n = []
 
-    for c in x:
-        idx = np.where(contrast1 == c)
-        n.append(float(len(idx[0])))
-        answer1 = len(np.where(answers[idx[0]] == 1)[0])
-        y.append(answer1 / n[-1])
-    return x, y, n
+def hz_to_midi(frequencies):
+    """Get MIDI note number(s) for given frequencies
 
-
-def cumgauss(x, mu, sigma):
-    """
-    The cumulative Gaussian at x, for the distribution with mean mu and
-    standard deviation sigma.
+    Examples
+    --------
+    >>> hz_to_midi(60)
+    34.506
+    >>> hz_to_midi([110, 220, 440])
+    array([ 45.,  57.,  69.])
```

Comment:
Usually `Examples` goes after params and returns, but i know that librosa is not consistent about this :disappointed: